### DISCUSSION TITLE
Paged scrollable area

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#2276] [Feature requests]
Hi.
I want to have such a component for usage in components with overflow problems, for example:
- mantine components: `Tabs, SegmentedControl, Breadcrumbs` - can be optionally used inside
- own components

I like how material tabs (in angular framework) controls overflow with custom slider nav button (prev, next), but now it is impossible to make by myself with mantine tabs, even if I created `paged-slider` for controls this logic.
Of course, you improved tabs 2 days ago (added flex-wrap: wrap), but in some business cases it is a bad practice.

Example of realization: https://github.com/ekerik220/paged-slider
Example of use in production
https://market.yandex.ru/catalog--televizory-i-aksessuary/24422170/list?searchContext=express&local-offers-first=0&priceto=999999&delivery-interval=5&glfilter=13887626%3A13891828%2C13899071

<img width="1494" alt="image" src="https://user-images.githubusercontent.com/18473137/187303702-9b35d457-5030-4e8b-896b-d3104432679c.png">

**Comment by rtivital** [comment]
This seems to be just a Carousel with dragFree option – https://mantine.dev/others/carousel/#cards-carousel

**Comment by 7iomka** [comment]
@rtivital Technically - yes. But the main purpose of this topic is to underline the main problem - we have multiple components, mentioned above that have requirements for layout / inner elements and this make impossible to use it with some sliders solutions (I like for example `swiper`, but I also mentioned a more simplified solution `paged-slider` but I have no ideas how to use it for example with Tabs / Breadcrumbs? SegmentedControl doesn't have children control at all.
I would like to hear your ideas)
